README.rst

```diff
@@ -75,24 +75,23 @@ Every time you develop something in this repo
 Deploying
 =========
 
-TODO: How can a new user go about deploying this component? Is it just a few
-commands? Is there a larger how-to that should be linked here?
+The Open edX Sink Clickhouse component is a django plugin which doesn't
+need independent deployment. Therefore, its setup is reasonably
+straightforward. First, it needs to be added to your service
+requirements, and then it will be installed alongside requirements
+of the service.
 
-PLACEHOLDER: For details on how to deploy this component, see the `deployment how-to`_
-
-.. _deployment how-to: https://docs.openedx.org/projects/openedx-event-sink-clickhouse/how-tos/how-to-deploy-this-component.html
+If the service you intend to use is either the LMS or CMS, then
+the library is installed alongside their requirements since the
+Nutmeg release.
```

Comment:
Can we add something specific about the settings that need to be set for this to work here?

Reply:
That paragraph should be removed, and I will add the settings